custom_model_runner/datarobot_drum/drum/typeschema_validation.py

```diff
@@ -316,6 +316,18 @@ def validate(self, dataframe):
 class NumColumns(BaseValidator):
     def __init__(self, condition, values):
         super(NumColumns, self).__init__(condition, values)
+        # check that all values are >=0
+        if not all([v >= 0 for v in values]):
+            raise ValueError("The value for number of columns can not be negative")
+        # check cases where value can be 0
```

Comment:
remove the line comment

Reply:
perhaps pull these out to static private methods with descriptive names like:
- assert_all_values_gte_zero
- assert_only_allowed_conditions_contain_zero_values